main.go

```diff
@@ -74,6 +75,7 @@ func getConfig() (*config, error) {
 			"deviceadm",
 			"deviceauth",
 			"inventory",
+			"inventory-enterprise",
```

Comment:
Once we are here: could you add also `create-artifact-worker`?

I was just about to prepare a new PR, but if you don't mind better to get it here to not conflict on your build in the server.

Reply:
And `workflows` also...

Reply:
Sure.

The bot is failing to sync, because there is no branch to sync to though. 
The new enterprise repos need feature branches present also. Not just master.

Reply:
I would leave create-artifact-worker out, we don't plan to have an enterprise version of it.

Reply:
Two different topics in this discussion, I think.

This list is about repos for which to run integration tests: hence all open source and enterprise repos go here.

Ole's comment about enterprise repos not having feature branches I don't understand. Do you mean release branches? As we discussed privately, integration-enterprise does not have release branches so it is okey that fails to sync PRs pointing to such branches, but it should work for master.

Reply:
Thank you @mender-test-bot ;)